net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,9 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+int kNonAddableResourceHeadersSize = 12;
+StringPieceVector kNonAddableResourceHeaders(kNonAddableResourceHeadersSize);
```

Comment:
Sorry to be a pain; this needs to be a pointer which you allocate in InitNonAddableResourceHeaders and delete in Terminate, so you don't run a static initializer.